Makefile-docker

```diff
@@ -1,5 +1,7 @@
 export PIP_COMMAND=pip3
-export PYTHON_COMMAND=python3
+export PYTHON_COMMAND=
```

Comment:
eek, I didn't mean to change this line.  It's probably what caused the circleci fail